platform-test/src/main/java/org/killbill/billing/platform/test/PlatformDBTestingHelper.java

```diff
@@ -57,12 +58,22 @@ protected PlatformDBTestingHelper() {
         if ("true".equals(System.getProperty("org.killbill.billing.dbi.test.h2"))) {
             log.info("Using h2 as the embedded database");
             instance = new H2EmbeddedDB();
+        } else if ("true".equals(System.getProperty("org.killbill.billing.dbi.test.postgresql"))) {
+            if (isUsingLocalInstance()) {
+                log.info("Using postgresql local database");
+                final String databaseName = System.getProperty("org.killbill.billing.dbi.test.localDb.database", "postgres");
+                final String username = System.getProperty("org.killbill.billing.dbi.test.localDb.username", "postgres");
+                final String password = System.getProperty("org.killbill.billing.dbi.test.localDb.password", "postgres");
+                instance = new PostgreSQLStandaloneDB(databaseName, username, password);
+            } else {
+                throw new UnsupportedOperationException("PostgreSQL can be chosen for stand-alone mode; set org.killbill.billing.dbi.test.useLocalDb to true.");
```

Comment:
I looked into the recommendation of airlift (https://github.com/killbill/killbill/issues/273) but was not able to personally implement this, as I'm on Windows and it's Mac-specific.  I do not believe it would take much effort to implement airlift.